providers/class-two-factor-totp.php

```diff
@@ -142,7 +142,7 @@ public function user_two_factor_options( $user ) {
 			</p>
 		<?php else : ?>
 			<p class="success">
-				<?php esc_html_e( 'Secret key configured and registered.', 'two-factor' ); ?>
+				<?php esc_html_e( 'Secret key is configured and registered. It is not possible to view it again for security reasons.', 'two-factor' ); ?>
```

Comment:
This has been a frequent question in the support forums so we explain why the secret key is not shown again.